Flashcards/script.js

```diff
@@ -133,9 +136,8 @@ let id = Questions[Math.floor(Math.random() * Questions.length)]
 
 next.addEventListener("click", () => {
 start = false;
-if (id < 4) {
+if (id < max) {
 id++;
```

Comment:
Nit: Try checking out the code formatting feature of VS Code.  You can access it by `Ctrl + Shift + P`.  This will help keep the code formatted in a way that is easier to read as your project grows. 
https://code.visualstudio.com/docs/editor/codebasics#_formatting

When you start using NPM/Yarn to manage dependencies for JavaScript projects, check out Prettier https://prettier.io/